src/singleinstance.py

```diff
@@ -75,7 +75,7 @@ def lock(self):
                     fcntl.lockf(self.fp, fcntl.LOCK_EX | fcntl.LOCK_NB)
                 self.lockPid = os.getpid()
             except IOError:
-                print 'Another instance of this application is already running'
+                print('Another instance of this application is already running')
                 sys.exit(-1)
```

Comment:
`sys.exit('Another instance of this application is already running')`?

Reply:
Possible but in that case return value can't be specified. If nothing relies on particular return values then exit('') is probably better.

Reply:
Only two values are significant here: 0 and any other. When you pass a string it's interpreted as program error, like any non-zero exit code.

Reply:
I guess nothing relies on specific values of non-zero so exit('') is fine.
```
$ python -c 'exit(5)'; echo $?
5
$ python -c 'exit(6)'; echo $?
6
$ python -c 'exit("error")'; echo $?
error
1
```